CI/harvester-installer/roles/jenkins/tasks/install_jenkins_on_Debian.yml

```diff
@@ -3,7 +3,7 @@
         Install prerequisite packages for Jenkins
   apt:
     name: [ca-certificates, openjdk-11-jdk, curl, apt-transport-https, gnupg,
-           python3, python3-pip, figlet]
+           python3, python3-pip, figlet, sshpass]
```

Comment:
Where is sshpass being used?

Reply:
I've removed that as it's not "needed" for the fix, that's more of the future state dependency for the air-gap work.
So as a separation of concerns, I've cleaned that up :smile: :+1:  good catch! @noahgildersleeve